integration-tests/integration-tests-quarkus/src/test/java/org/kie/kogito/integrationtests/quarkus/EnumsTest.java

```diff
@@ -65,27 +66,27 @@ void testSubmitMovie() {
             .extract()
                 .path("id");
 
-        Map<String, String> tasks = given()
-                .when()
-                    .get("/cinema/{pid}/tasks", pid)
-                .then()
-                    .statusCode(200)
-                .extract()
-                    .body()
-                    .as(Map.class);
-        assertEquals(1, tasks.size());
+        WorkItem[] tasks = given()
+            .when()
+            .get("/cinema/{pid}/tasks", pid)
+            .then()
+            .statusCode(200)
+            .extract()
+            .body()
+            .as(TestWorkItem[].class);
+        assertEquals(1, tasks.length);
```

Comment:
My suggestion to simplify this without need for creating the TestWorkItem class 
```suggestion
        String taskId = given()
                .when()
                .get("/cinema/{pid}/tasks", pid)
                .then()
                .statusCode(200)
                .body("$.size", is(1))
                .body("[0].name", is("ReviewRatingTask"))
                .extract()
                .path("[0].id");

        Map<String, Object> reviewedRating = new HashMap<>();
        reviewedRating.put("reviewedRating", Rating.PG_13);
        given()
            .contentType(ContentType.JSON)
            .when()
            .body(reviewedRating)
            .post("/cinema/{pid}/ReviewRatingTask/{taskId}", pid, taskId)
            .then()
            .statusCode(200)
            .body("movie.rating", equalTo(Rating.PG_13.name()));
```

Reply:
I prefer to deserialize the whole workitem id to ensure they match.

Reply:
why not take advantage of RestAssured capabilities to check that the request actually matches using `.body("[0].attribute")` for that? I find that easier to read and maintain, but that is just my point of view.

Reply:
Because I also wanted to check that the response do not contain any extra attribute that is not contemplated in WorkItem interface. 
If workItem interface is changed, the test needs to be changed too, and using WorkItem this will be enforced by the compiler.

Reply:
yep, I see your point but I would be more concerned in checking that the attribute values returned are the expected ones, so if something changes as in value is not expected and attribute is removed, that would break the test. But adding a new attribute isn't much of concern I would say, we should be able to evolve the model without breaking existing code. That's why I think we have different ways to tackle this, but completely fine to have different opinions and approach right :)

Reply:
Ok, since we actually have two similar test, one for quarkus and one for springboot, lets took the salomonic approach, one will check only expected attributes (quarkus), the other (springboot) will check the whole item